src/sk_import.jl

```diff
@@ -0,0 +1,66 @@
+
+"""
+Extract all symbols from an expression
+"""
+_symbols_in(e::Expr) = union(_symbols_in(e.head), map(_symbols_in, e.args)...)
+_symbols_in(e::Symbol) = Set([e])
+_symbols_in(::Any) = Set()
+
+
+"""
+Check if function has a julia wrapper. This occurs
+when data needs to be prepared before being passed to the
+python function or processed after it is returned. 
+"""
+function _julia_wraper(translated_modules, mod, what)
```

Comment:
maybe you mean "wrapper" instead of "wraper"

Reply:
Updated